health/notifications/custom/metadata.yaml

```diff
@@ -51,8 +52,9 @@
             description: "You can look at the other senders in `/usr/libexec/netdata/plugins.d/alarm-notify.sh` for examples of how to modify the function in this configuration file."
             required: false
             detailed_description: >
```

Comment:
I think `detailed_description` is almost always will be `|` (pre-formatted text)